src/Components/Productpage/ProductListing/ProductList.jsx

```diff
@@ -0,0 +1,53 @@
+import Product from "./Product";
```

Comment:
Appreciation: Really great logic for the paginated page loved it.

Reply:
Thank youuu.